dbt/scripts/run_iasworld_data_tests.py

```diff
@@ -1087,9 +1114,15 @@ def get_test_cache_path(
     manifest_filepath: str,
     townships: typing.Tuple[str],
 ) -> str:
-    """Return the path to the cache where test results are stored.
-    The `run_results_filepath` and `manifest_filepath` are used to generated
-    a hash key that uniquely defines the cache key for a given test run."""
+    """Return the path to the cache where test results are stored, or an
+    empty string if no cache exists yet. The `run_results_filepath` and
+    `manifest_filepath` are used to generated a hash key that uniquely defines
+    the cache key for a given test run."""
+    if not os.path.isfile(run_results_filepath) or not os.path.isfile(
+        manifest_filepath
+    ):
+        return ""
```

Comment:
I'm not super confident in this interface, although I don't know if getting it exactly right is high priority. One alternative would be to raise an error if one or both of these files is missing, since we should only ever call this function under one of two conditions:

1. The caller wants to read from the cache for a previous run and has instructed the script to do so by passing in the `--use-cached` param, in which case we the script is already configured to error if the cache does not exist
2. The script wants to write to the cache for the most recent run, in which case it will have always already completed a dbt run in the immediate past

Reply:
I think that's fine since it's just for local testing purposes!